### ISSUE TITLE
Away Profile

### ISSUE THREAD

**Issue by Emmitts89** [Project-Caruso/MessagePro#12]
Away profile sent message to itself in a thread that never received a text

**Comment by jcaruso001** [comment]
You seem to be correct on this, let me investigate more because I just sent a message to you while my away profile was up...

**Comment by jcaruso001** [comment]
This should be good.

**Comment by Emmitts89** [comment]
When the away message is on, it does automatically send when you get an incoming message, but it does not show as a new message for the phone where the profile is on.

**Comment by jcaruso001** [comment]
Let me rephrase to make sure i understand. If you have a profile active your not receiving any new messages? It is that way by design, you should not receive messages while your profile is on, you receive those messages when you turn your profile off. Thus creating the un-interrupted effect you wished for.

**Comment by Emmitts89** [comment]
This makes sense as to how/why you did that. My idea for how it would work would be that you would still get the message in your inbox and have it shown as unread, but it would not give you a notification(pop up or vibrate your phone) until the profile was turned off. So if I turn the profile off, then you would get the blue dots (shown as unread and notification for the messages received when the profile was on? Would the threads update while the profile is on and move to the top of the order of conversation threads (i.e. the conversations with the most recent messages on the top of the main message screen)?

**Comment by jcaruso001** [comment]
We need to put up a "Lightbox" that says a n away profile is active and overlay it on the chat activities thread.

**Comment by jcaruso001** [comment]
Please verify the overlay looks good when your away profile is active.

**Comment by Emmitts89** [comment]
This looks good but is there a way to put it across the top bar so that the user could still look at their messages in case they aren't ready to turn it off?

**Comment by jcaruso001** [comment]
No, because then i'd have to add it more logic to handle them attempting to send messages while their in away mode. If you want to change this after v1 we can re-visit that.
